pages.pl/common/fossil-ci.md

```diff
@@ -1,4 +1,4 @@
-# fossil-ci
+# fossil ci
 
 > To polecenie jest aliasem `fossil-commit`.
```

Comment:
Hyphens haven't been removed from descriptions. This is also the case with other languages.
```suggestion
> To polecenie jest aliasem `fossil commit`.
```